sdk/vc-data/src/data/experience/v1.test.ts

```diff
@@ -0,0 +1,140 @@
+import { VCExperiencePersonV1, VCSExperiencePersonV1, getVCExperiencePersonV1Context } from './v1'
+import { expandVC } from '../../testUtil.test'
+
+describe('VCExperiencePersonV1', () => {
+  it('expands correctly', async () => {
+    expect.assertions(1)
+
+    const expanded = await expandVC<VCExperiencePersonV1, VCSExperiencePersonV1>({
+      type: 'ExperienceCredentialPersonV1',
+      data: {
+        '@type': ['Person', 'PersonE', 'ExperiencePerson'],
+        name: 'Bob Belcher',
+        officialEmail: 'bob.b@affinidi.com',
```

Comment:
please add personal email as well. since employees receiving this VC will not be a part of the company